### ISSUE TITLE
`__qualname__` mistakenly believed to be defined as a global

### ISSUE THREAD

**Issue by Sachaa-Thanasius** [microsoft/pyright#6699]
**Describe the bug**
Pyright doesn't flag `__qualname__` as undefined when it's used on the global scope, but running code with that assumption causes a `NameError`.

**Code or Screenshots**
```python
val = __qualname__
```
Pyright gives no indication this is invalid.

Result when run:
```py
Traceback (most recent call last):
  File "/home/thanos/Documents/projects/scratch2.py", line 1, in <module>
    val = __qualname__
          ^^^^^^^^^^^^
NameError: name '__qualname__' is not defined
```

**VS Code extension or command-line**
pyright 1.1.339

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.340, which I just published. It will be included in a future release of pylance.
